testsuite/features/support/twopence_init.rb

```diff
@@ -323,10 +328,11 @@ def client_public_ip(host)
 
   %w[br0 eth0 eth1 ens0 ens1 ens2 ens3 ens4 ens5 ens6].each do |dev|
     output, code = node.run("ip address show dev #{dev} | grep 'inet '", check_errors: false)
```

Comment:
As apparently we could have error codes here, let's add `verbose: true` optional parameter to the method please.

Reply:
Done.

Reply:
Due to the enabled verbose flag we always get the following now:

```bash
(...)                                                                                                     
ip address show dev br0 | grep 'inet ' returned status code = 1.
Output:                                                                         
Device "br0" does not exist.                                                                                                                                    
ip address show dev eth0 | grep 'inet ' returned status code = 0.                                                                                               
Output:                                                                                                                                                         
    inet 192.168.122.41/24 brd 192.168.122.255 scope global eth0                                                                                                
ip address show dev br0 | grep 'inet ' returned status code = 1.                                                                                                
Output:                                                                                                                                                         
Device "br0" does not exist.                                                                                                                                    
ip address show dev eth0 | grep 'inet ' returned status code = 0.
Output:                                                                                                                                                         
    inet 192.168.122.31/24 brd 192.168.122.255 scope global eth0                                                                                                
ip address show dev br0 | grep 'inet ' returned status code = 0.
Output:                                                                                                                                                         
    inet 192.168.122.36/24 brd 192.168.122.255 scope global br0                                                                                                 
ip address show dev br0 | grep 'inet ' returned status code = 0.                                                                                                
Output:                                                                                                                                                         
(...)
```